conformance_tests/sysman/test_sysman_power/src/test_sysman_power.cpp

```diff
@@ -490,35 +490,39 @@ TEST_F(
         }
       }
 
-      if (power_sustained_set.limitValueLocked == false) {
-        lzt::set_power_limits_ext(
-            p_power_handle, &count_power,
-            power_limits_descriptors.data()); // set power limits for all
-                                              // descriptors
-
-        zes_power_limit_ext_desc_t power_sustained_get = {};
-
-        auto power_limits_descriptors_get =
-            lzt::get_power_limits_ext(p_power_handle, &count_power);
-        for (auto p_power_limits_descriptor_get :
-             power_limits_descriptors_get) {
-          if (p_power_limits_descriptor_get.level ==
-              ZES_POWER_LEVEL_SUSTAINED) {
-            power_sustained_get = p_power_limits_descriptor_get;
+      if (power_sustained_set.level == ZES_POWER_LEVEL_SUSTAINED) {
```

Comment:
Could you please share what this change is fixing? Could you attach a related to jira?

Reply:
In this test a empty structure is 0 initialized.
`zes_power_limit_ext_desc_t power_sustained_set = {};`

It then iterates over all power limit descriptors and tries to find the ZES_POWER_LEVEL_SUSTAINED limit descriptor to fill `power_sustained_set ` with.

The problem here was that there are some platforms where  ZES_POWER_LEVEL_SUSTAINED will not be available and `power_sustained_set ` will be empty. Hence we should first check if it really has data for ZES_POWER_LEVEL_SUSTAINED before proceeding with rest of the test.

There is no Jira for this as I just observed them failing on elmo for long time and fixed them.

Reply:
So how could we validate condition where on supported platforms ZES_POWER_LEVEL_SUSTAINED will not be available?

Reply:
I don't think there is a way to know available limits before hand and configure CTS accordingly for different platforms.

Reply:
@AshwinKumarKulkarni @saik-intel @vishnu-khanth I feel this validation could be done in embargo tests.